src/sql/parts/notebook/outputs/common/nbformat.ts

```diff
@@ -0,0 +1,499 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+// Notebook format interfaces
+// https://nbformat.readthedocs.io/en/latest/format_description.html
+// https://github.com/jupyter/nbformat/blob/master/nbformat/v4/nbformat.v4.schema.json
+
+
+import { JSONObject } from './jsonext';
+import { nb } from 'sqlops';
+
+/**
+ * A namespace for nbformat interfaces.
+ */
+export namespace nbformat {
```

Comment:
We should consider removing this file and recompiling against the `sqlops.nb` types. I think we can check in as-is for now but please create a task to do this (I think there'll be some annoying gaps so we should do as its own review).

Reply:
Done